packages/macros/tests/fixtures/define-model/vue2-custom-vmodel.vue

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+export default {
```

Comment:
Maybe we should handle another situation: export default with `defineComponent` function.

```ts
export default defineComponent({
  model: {
    prop: 'value',
    event: 'changeInput'
  },
})
```